acceptance/bundle/templates/dbt-sql/script

```diff
@@ -3,3 +3,5 @@ trace $CLI bundle init dbt-sql --config-file ./input.json --output-dir output
 cd output/my_dbt_sql
 trace $CLI bundle validate -t dev
 trace $CLI bundle validate -t prod
+
+mv .gitignore out.gitignore
```

Comment:
Could you include the intent behind this as a comment?

Reply:
Added.

Reply:
Could you add the PR URL instead of the number?

Reply:
Could you also add more context here? That the `.gitignore` is generated by the CLI and thus is "output"?

Reply:
> Any particular reason to prefer full URLs?

It can not be clear which repo the PR number refers to. While implied with a number, a URL is more explicit. You can also cmd+click it which does not work with #<number>. 

Either is fine, though, just me being nitpicky.